_pages/plugins/Nscale-segmentation.md

```diff
@@ -0,0 +1,47 @@
+---
+mediawiki: Nscale_segmentaion
+name: "Nscale_segmentaion"
+title: Nscale_segmentaion
+categories: [Segmentation]
+website: "https://sites.imagej.net/Nscale_segmentaion/"
+dev-status: "stable"
+Author: ['@Eden Tamrat']
+---
+
+The watershed algorithm and the segmentation tools within the Fiji software often dont distinguish a cluster of nuclei that are in close proximity, erroneously grouping them together. This leads to an inaccurate analysis of average nuclear size. To address this issue, this customizable segmentation plugin employs a scale parameter to calibrate the image, to give a more precise separation of touching nuclei.
+
+## Instalation
+
+To instal the plugin onto Fiji first copy the script from the update site (https://sites.imagej.net/Nscale_segmentaion/) and save it as 
+a txt. (Open notebook, paste script onto notebook and save it). To run it go to plugins>macros>install> and click on the Nscale_segmentaion script you just saved. The plugin should show up like this: 
+
+![image](https://github.com/EdenTamrat/imagej.github.io/assets/149198009/f5814cf6-2358-41ec-9cbb-1c3547055c9d)
```

Comment:
These images need to be uploaded as part of this pull request. I believe you can drag and drop them onto your browser tab while viewing this PR. Or if you have trouble, you can share them via some cloud service temporarily so that I can download them and add them here myself. Whatever is easier!